chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1955,14 +1954,14 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> List[CoinRecord]:
         # expect both test items to get included in the block.
         # NOTE: The expected block cost is smaller than the sum of items' costs
         # because of the spend bundle aggregation that creates the block bundle.
-        (TEST_FILL_RATE_ITEM_COST * 2, 2, TEST_FILL_RATE_ITEM_COST * 2 - 61_829),
+        (TEST_FILL_RATE_ITEM_COST * 2, 2, TEST_FILL_RATE_ITEM_COST * 2 - 156_020),
         # Here we set the block cost limit to twice the test items' cost - 1,
         # so we expect only one of the two test items to get included in the block.
         (TEST_FILL_RATE_ITEM_COST * 2 - 1, 1, TEST_FILL_RATE_ITEM_COST),
     ],
 )
 async def test_fill_rate_block_validation(
-    consensus_mode: ConsensusMode,
```

Comment:
apparently we did not get a warning on this unused parameter. is this still exempt from mypy?